docs/opentelemetry.md

```diff
@@ -0,0 +1,23 @@
+# OpenTelemetry support
+
+kOps is experimenting with initial support for OpenTelemetry, starting with tracing.
+
+The support should be considered experimental; the trace file format and schema will likely change, and these initial experiments might be removed entirely.
+
+kOps supports a "serverless" mode of operation, where log the OpenTracing output to a file.  We do this because our e2e test runner (prow) doesn't yet have a destination for OpenTelemetry data.
```

Comment:
```suggestion
kOps supports a "serverless" mode of operation, where it logs the OpenTracing output to a file.  We do this because our e2e test runner (prow) doesn't yet have a destination for OpenTelemetry data.
```